precompiles/common/types.go

```diff
@@ -30,6 +30,15 @@ var (
 	}
 )
 
+// Allocation defines the spend limit for a particular port and channel
+// we need this to be able to unpack to big.Int instead of sdkmath.Int
```

Comment:
```suggestion
// Allocation defines the spend limit for a particular port and channel.
// We need this to be able to unpack to big.Int instead of sdkmath.Int.
```